modules/express/src/fetchEncryptedPrivKeys.ts

```diff
@@ -0,0 +1,46 @@
+import { writeFile } from 'fs';
+import { BitGo } from 'bitgo';
+
+// TODO: set env to 'test' or 'prod'
+const bg = new BitGo({ env: 'test' });
+// TODO: set your access token here
+const accessToken = '';
+// TODO: set your coin type and wallet ids here e.g.
+// const walletIds = [
+//   ['tbtc', '61f039aad587c2000145c687373e0fa1'],
+//   ['tgeth', '61fb21819c14dd000755f8de3a18e461']
+// ];
+const walletIds = [];
+
+type Output = {
+  [key: string]: string
+}
+
+async function main() {
+  bg.authenticateWithAccessToken({ accessToken });
+
+  // get the encrypted user privKey for each walletId and store in the JSON output
+  const output: Output = {};
+  for (let i = 0; i < walletIds.length; i++) {
+    const coin = walletIds[i][0];
+    const id = walletIds[i][1];
+    const wallet = await bg.coin(coin).wallets().get({ id });
+    const userKeyId = wallet.keyIds()[0];
+    const keychain = await bg.coin(coin).keychains().get({ id: userKeyId });
+    if (keychain.encryptedPrv === undefined) {
+      throw new Error(`could not find a ${coin} private key for wallet id ${id}`);
+    }
+    output[id] = keychain.encryptedPrv;
+  }
```

Comment:
```suggestion
  for (const [coin, walletIds] of Object.entries(walletIds)) {
    const coin = bg.coin(coin);
    for (const id of walletIds) {
      const wallet = await coin.wallets().get({ id });
      const userKeyId = wallet.keyIds()[0];
      const keychain = await coin.keychains().get({ id: userKeyId });
      if (keychain.encryptedPrv === undefined) {
        console.warn(`could not find a ${coin} encrypted user private key for wallet id ${id}, skipping`);
        continue;
      }
      output[id] = keychain.encryptedPrv;
    }
  }
```

Reply:
thanks, this is cleaner. I implemented this but used new names `coin`/`coinName` and `walletIds`/`ids` to avoid the names conflicting.